resources/assets/js/application.js

```diff
@@ -1,34 +1,44 @@
 window.global ||= window;
 // Vendors
 import SignaturePad from "signature_pad";
+
 global.SignaturePad = SignaturePad;
 import moment from "moment/moment.js";
+
 global.moment = moment;
 import quagga from 'quagga';
+
 global.Quagga = quagga;
 
 import './countdown-timer'
 import './utilities'
 import './broto'
 import './nightMode'
 // Execute theme JavaScript
-window[config.theme]?.()
+if ((new Date().getMonth() + 1) !== 12) {
```

Comment:
Why add this check here?

Reply:
Otherwise you will still get all the stars from nightmode and all the brotos from the broto theme, it would be best if this check could know the theme but I do not think we have acces to that in the JS only files.

Reply:
No the needed acces is to the disable-december cookie but that does not get to the js. When I console.log(document.cookie) it does not show up, even when set

Reply:
Sure, I guess we revert this code after December anyways.